src/Fable.Transforms/FSharp2Fable.Util.fs

```diff
@@ -1192,6 +1192,10 @@ module Util =
             else
                 // Indexed properties keep the get_/set_ prefix and are compiled as methods
                 if indexedProp then memb.CompiledName, false, false
+                // performance optimization, compile get_Current as instance call instead of a getter
+                elif memb.FullName = "System.Collections.IEnumerator.get_Current" ||
+                     memb.FullName = "System.Collections.Generic.IEnumerator.get_Current"
```

Comment:
I would prefer having `System.Collections(.Generic).IEnumerator` mangled as the rest of interfaces from the BCL, as mangled properties are already always compiled as methods. That way we will avid avoid having to add an exception to an exception.

Reply:
Sure, if the mangled IEnumerator methods are consistent enough to be called from Seq.ts.

Reply:
That is, until eventually we convert Seq module to F#.

Reply:
Most of the functions in Seq.ts rely on JS iterators, so I guess we will only have to change `Seq/toIterator`

Reply:
Yes, any idea what the mangled IEnumerator method names should look like?

Reply:
Seems the generic version is already mangled. If you comment out [this line](https://github.com/fable-compiler/Fable/blob/3a24934a73b18504e5ad417e490fc691bc534497/src/Fable.Transforms/FSharp2Fable.Util.fs#L1171), compile fable library (`npm run build library`) and check `class MapTreeModule_mkIEnumerator$0027$2` you'll see the generated names are:

```
    ["System.Collections.Generic.IEnumerator`1.get_Current"]() {
    ["System.Collections.IEnumerator.get_Current"]() {
    ["System.Collections.IEnumerator.MoveNext"]() {
    ["System.Collections.IEnumerator.Reset"]() {
```

Reply:
@alfonsogarciacaro OK, fixed per your recommendation.